docs/back-end-api/erc20CollateralToken/graphql.md

```diff
@@ -644,14 +615,12 @@ Upon successful request completion, the server will issue a status code of 200 a
 
 ### `Lend`
 
-Lend to a pool.
+Lend to a pool the specific `usdc` amount.
```

Comment:
not only usdc, but the based token contract, usdc is an example but it could be euroc or any other that has a precision of 6 and follows erc20 token standart